BSB_lan_EEPROMconfig.h

```diff
@@ -107,19 +108,20 @@ typedef struct {
 PROGMEM_LATE const category_list_struct catalist[]={
   {CCAT_GENERAL,        CAT_GENERAL_TXT},
   {CCAT_IPV4,           CAT_IPV4_TXT},
-  {CCAT_MQTT,           CAT_MQTT_TXT}
+  {CCAT_MQTT,           CAT_MQTT_TXT},
+  {CCAT_BUS,            CAT_BUS_TXT}
 };
```

Comment:
Oh great, thanks! I can think about the category names in German and English and make a list, so that you can add them? 
Would be much better than me messing around here.. ;)

Reply:
Sure!

Reply:
For 'Sensors' (=English) it should be 'Sensoren' in German btw, but I'll make the list for the categories and the German/English expressions, so that you can do it all in one process.